pandas/io/excel/_base.py

```diff
@@ -123,6 +123,10 @@
          then `odf <https://pypi.org/project/odfpy/>`_ will be used.
        - Otherwise if ``path_or_buffer`` is an xls format,
          ``xlrd`` will be used.
+       - Otherwise if ``path_or_buffer`` is in xlsb format,
+         ``pyxlsb`` will be used.
```

Comment:
Let's link to `` `pyxlsb <https://pypi.org/project/pyxlsb/>`_ `` here, as is with the other engines.